pkg/buildchange/lifecycle_change.go

```diff
@@ -0,0 +1,52 @@
+package buildchange
+
+import (
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/pkg/errors"
+
+	buildapi "github.com/pivotal/kpack/pkg/apis/build/v1alpha2"
+)
+
+func NewLifecycleChange(oldLifecycleImageRefStr, newLifecycleImageRefStr string) Change {
```

Comment:
This is basically the same as `NewStackChange`. But, I made a new function because 🤷🏼